lib/menu.js

```diff
@@ -82,90 +88,80 @@ MenuItem.prototype.get = function (roles, path) {
   if (path) {
     if (!path.length) return this;
     var n = path.shift();
-    var index = list.indexOf (n);
-    return this.submenus[index] ? this.submenus[index].get(roles,path) : undefined;
+    var index = list.indexOf(n);
+    return this.submenus[index] ? this.submenus[index].get(roles, path) : undefined;
   }
 
-  if(!MenuItem.hasRole('admin', roles)) {
+  if (!MenuItem.hasRole('admin', roles)) {
     if (this.roles) {
       if (!_.intersection(this.roles, roles).length) return undefined;
     }
   }
-
-  return new MenuItem ({
+  return new MenuItem({
     roles: this.roles || null,
-    link : this.link || null,
-    title:this.title || null,
-    name : this.name || null,
-    icon : this.icon || null,
-    submenus : this.submenus.map(get_get.bind(null, roles)).filter(remove_nulls),
+    link: this.link || null,
+    title: this.title || null,
+    name: this.name || null,
+    options: this.options || null,
+    submenus: this.submenus.map(get_get.bind(null, roles)).filter(remove_nulls),
```

Comment:
Identifier 'get_get' is not in camel case.<br>Identifier 'remove_nulls' is not in camel case.